Sources/MapGL/Model/Padding.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/// Padding in density independent pixels from the different sides of the map canvas.
```

Comment:
То есть в логических точках?

```suggestion
/// Padding in points from the different sides of the map canvas.
```